requirements.txt

```diff
@@ -1,2 +1,3 @@
 mako
 PyYAML
+nose
```

Comment:
Ooh, interesting. Are you proposing that I move the `nose` requirement to a different file? Or should I only comment it within `requirements.txt`?